src/pages/EditorPage/EditorPanel.jsx

```diff
@@ -617,6 +627,20 @@ const EditorPanel = ({
                       widthExpand
                     />
                   )
+                    } else if (field === 'tags') {
+                      input = (
+                    <TagsSelect
+                      value={isMultiple ? union(...isMultiple) : value || []}
+                      tags={projectTagsObject}
+                      tagsOrder={projectTagsOrder}
+                      isMultiple={!!isMultiple}
+                      onChange={(v) => handleLocalChange(v, changeKey, field)}
+                      align="right"
+                      styleDropdown={{ overflow: 'hidden' }}
+                      width={200}
```

Comment:
Can we please add a border like the rest of the fields.

```
style={{
  border: isChanged
    ? '3px solid var(--md-sys-color-primary)'
    : '1px solid var(--md-sys-color-outline-variant)',
}}
```

Reply:
Sorry this piece of code seems to be wrong, my bad. The border shouldn't be changing here but the background should be going blue.

This is broken for status select and tags select. Look at how the **Type** select dropdown background goes to blue `var(--md-sys-color-primary)` and the inside text goes to dark blue `var(--md-sys-color-on-primary)`. 

![image](https://github.com/ynput/ayon-frontend/assets/49156310/008d7436-b92f-4ef9-bf12-f31bfd3a689c)

We need to do the same for both **Status** and **Tags**. **Tags**

![image](https://github.com/ynput/ayon-frontend/assets/49156310/a05a12c5-1bef-47c3-bba5-2f23963e26d2)

There is a chance there is something broken with the ARC component but lets just use `style` for now and fix ARC later.

Reply:
I have changed styles as you suggested.

After looking into ayon-react-components I thought that It would be better to put styles into `buttonStyle` property instead of `style` + I added the isChanged property to make it work.